cpp/third_party/CMakeLists.txt

```diff
@@ -8,10 +8,6 @@ if(NOT ${ARCTICDB_USING_CONDA})
     add_subdirectory(rapidcheck/extras/gtest)
     add_subdirectory(Remotery)
 endif()
-add_subdirectory(lmdbcxx)
-add_subdirectory(Remotery)
 
+add_subdirectory(lmdbcxx)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/Remotery/lib)
```

Comment:
This is also dangerous: Remotery from Conda can have different headers from the submodule in this repo.

Reply:
I'll move inside the conditional and see if Conda continues to build.